src/app/containers/ProposalRow/index.tsx

```diff
@@ -134,7 +128,12 @@ export function ProposalRow({ proposal }: Props) {
           <>
             <td className="font-montserrat max-w-sm truncate">
               <Linkify newTab={true}>
```

Comment:
We have a duplicate on line `84`, pls extract it to a component (you can leave the new component in this file).